### PR TITLE
Attendance integration

### PR DESCRIPTION
**Pull request by santhuraja**
@sathyamus : Please review once and merge to the master

### CODE REVIEW

File: mobile-user-management/src/app/app.module.ts [hunk 1]
```diff
@@ -8,13 +8,20 @@ import { StatusBar } from '@ionic-native/status-bar/ngx';
 
 import { AppComponent } from './app.component';
 import { AppRoutingModule } from './app-routing.module';
-
 import {FormsModule, ReactiveFormsModule} from '@angular/forms';
 import {HttpClientModule} from '@angular/common/http';
+import { Camera, CameraOptions } from '@ionic-native/camera/ngx';
```
Comment (sathyamus) [inline mobile-user-management/src/app/app.module.ts:13]: Camera is integrated here ?

File: mobile-user-management/package.json [hunk 2]
```diff
@@ -13,55 +13,72 @@
...
+    "@ionic-native/status-bar": "^5.25.0",
+    "@ionic/angular": "^4.11.10",
+    "@ng-bootstrap/ng-bootstrap": "^6.1.0",
     "cordova-android": "7.1.4",
+    "cordova-browser": "^6.0.0",
     "cordova-ios": "5.0.1",
-    "cordova-plugin-device": "^2.0.2",
-    "cordova-plugin-ionic-keyboard": "^2.1.3",
+    "cordova-plugin-calendar": "^5.1.5",
+    "cordova-plugin-camera": "^4.1.0",
```
Comment (sathyamus) [inline mobile-user-management/package.json:40]: Are we using Camera for upload image ?

File: mobile-user-management/src/app/app.module.ts (hunk 1 above)
Comment (santhuraja) [reply mobile-user-management/src/app/app.module.ts:13]: Yes, it is opening camera. Not yet tested with backend

**Comment by sathyamus** [comment]
#9 , is having all this branch changes, hence closing this

**Pull request closed without merging**